plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -61,7 +61,7 @@ func NewSelectorSpreadPriority(
 
 // Returns selectors of services, RCs and RSs matching the given pod.
 func getSelectors(pod *v1.Pod, sl algorithm.ServiceLister, cl algorithm.ControllerLister, rsl algorithm.ReplicaSetLister, ssl algorithm.StatefulSetLister) []labels.Selector {
-	selectors := make([]labels.Selector, 0, 3)
```

Comment:
I checked with others as well, it seems its a old change. AFAIK, no its not going to reduced test coverage. Its kind of confusing why we have 3 in first place.(I think previously there were 3 selectors until stateful set was added recently as part of #41708).